cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -454,6 +457,15 @@ public boolean verify(String s, SSLSession sslSession) {
                 args = args.subList(2,args.size());
                 continue;
             }
+            if(head.equals("-v")) {
```

Comment:
I was not feeling it was worth it, there's only few messages which could be outputted and it's easier to have just a single parameter to turn logs on IMO.